rebuild-kyrix-server

```diff
@@ -1,11 +1,18 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 KV=${1:-adsah/kyrix:latest}
 
 sudo docker build . -f Dockerfile-kyrix-alpine -t $KV
 sudo docker push $KV
 
 echo "updating kyrix-deployment.yaml"
-sed -i "s@image:.*@image: $KV@" kyrix-deployment.yaml
+
+# https://stackoverflow.com/questions/3466166/how-to-check-if-running-in-cygwin-mac-or-linux
+# https://ed.gs/2016/01/26/os-x-sed-invalid-command-code/
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
good catch - kevin ran into this too. I'm surprised we can't pick one syntax for both and skip if darwin which seems ghetto and fragile.

Reply:
Looks like there is problem with `sed` in `redeploy-citus` as well, which is expected. It doesn't break things but it generates weird stuff like `orker-deployment.yaml` on my macbook.  

I think we should replace all `sed`s that modify local files with `perl`. I'm looking into it.

Reply:
I made a change, it worked on my mac. Can you help review and test it?